Makefile

```diff
@@ -46,8 +46,7 @@ lint:
 .PHONY: security
 security:
 	@echo Security...
-	@echo NOTE: Currently disabled due to https://github.com/jaegertracing/jaeger-operator/issues/379
-	@#gosec -exclude=G104 $(PACKAGES)
+	@gosec -quiet -exclude=G104 $(PACKAGES) 2>/dev/null
```

Comment:
Could you confirm that `gosec` is still able to block the PR in case of a violation? For instance, add some "dummy" code that introduces a new problem and see if `make security` returns the appropriate return code + message. 

I suspect the `2>/dev/null` might also be suppressing valuable output in case a violation is found.

Reply:
Running `make security` locally, after removing the `-exclude=G184` the output was:
```
$ make security
Security...
Results:
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/apis/io/v1alpha1/options.go:17] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > options.parse(o)
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/apis/io/v1alpha1/options.go:41] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > json.Unmarshal(b, &entries)
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:44] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("jaeger-version", cmd.Flags().Lookup("jaeger-version"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:47] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("jaeger-agent-image", cmd.Flags().Lookup("jaeger-agent-image"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:50] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("jaeger-query-image", cmd.Flags().Lookup("jaeger-query-image"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:53] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("jaeger-collector-image", cmd.Flags().Lookup("jaeger-collector-image"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:56] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("jaeger-ingester-image", cmd.Flags().Lookup("jaeger-ingester-image"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:59] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("jaeger-all-in-one-image", cmd.Flags().Lookup("jaeger-all-in-one-image"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:62] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("jaeger-cassandra-schema-image", cmd.Flags().Lookup("jaeger-cassandra-schema-image"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:65] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("jaeger-spark-dependencies-image", cmd.Flags().Lookup("jaeger-spark-dependencies-image"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:68] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("jaeger-es-index-cleaner-image", cmd.Flags().Lookup("jaeger-es-index-cleaner-image"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:71] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("jaeger-es-rollover-image", cmd.Flags().Lookup("jaeger-es-rollover-image"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:74] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("jaeger-elasticsearch-image", cmd.Flags().Lookup("jaeger-elasticsearch-image"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:77] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("openshift-oauth-proxy-image", cmd.Flags().Lookup("openshift-oauth-proxy-image"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:80] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("platform", cmd.Flags().Lookup("platform"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:83] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("es-provision", cmd.Flags().Lookup("es-provision"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:86] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("log-level", cmd.Flags().Lookup("log-level"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:89] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("metrics-host", cmd.Flags().Lookup("metrics-host"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/cmd/start/main.go:92] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > viper.BindPFlag("metrics-port", cmd.Flags().Lookup("metrics-port"))
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/storage/elasticsearch_secrets.go:214] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > os.RemoveAll(path)
[.../go/src/github.com/jaegertracing/jaeger-operator/pkg/strategy/controller.go:219] - G104: Errors unhandled. (Confidence: HIGH, Severity: LOW)
  > json.Unmarshal([]byte(menuStr), &menuArray)
Summary:
   Files: 86
   Lines: 7899
   Nosec: 1
  Issues: 21

make: *** [Makefile:49: security] Error 1
```